ice/src/main/java/com/altinity/ice/cli/internal/cmd/Describe.java

```diff
@@ -74,42 +71,18 @@ public static void run(RESTCatalog catalog, String target, boolean json, Option.
         if (targetTable != null && !targetTable.equals(tableId.name())) {
           continue;
         }
-        org.apache.iceberg.Table table = catalog.loadTable(tableId);
-        Snapshot snapshot = table.currentSnapshot();
-        Table.Snapshot snapshotInfo = null;
-        if (snapshot != null) {
-          snapshotInfo =
-              new Table.Snapshot(
-                  snapshot.sequenceNumber(),
-                  snapshot.snapshotId(),
-                  snapshot.parentId(),
-                  snapshot.timestampMillis(),
-                  Instant.ofEpochMilli(snapshot.timestampMillis()).toString(),
-                  Instant.ofEpochMilli(snapshot.timestampMillis())
-                      .atZone(ZoneId.systemDefault())
-                      .format(DateTimeFormatter.ISO_OFFSET_DATE_TIME),
-                  snapshot.operation(),
-                  snapshot.summary(),
-                  snapshot.manifestListLocation());
-        }
 
-        List<Table.Metrics> metrics = null;
-        if (optionsSet.contains(Option.INCLUDE_METRICS)) {
-          metrics = gatherTableMetrics(table);
+        Describe.Table.Data tableData = null;
+        Table.Metadata tableMetadata;
+        try {
+          tableData = gatherTableData(catalog, tableId, optionsSet);
+          tableMetadata = new Table.Metadata(tableId.toString());
+        } catch (Exception e) {
```

Comment:
if there is a temporary network issue (e.g. request time out) (or runtime exception due to a bug in gatherTableMetrics, et.c) it shouldn't probably not be returned as table status

Reply:
This is open for discussion.

I’d prefer consistent behavior and would include any error as part of the status. 
That way, the `describe` command wouldn’t fail immediately if only one of the tables has an issue.

For now, I’ve updated the catch clause to handle only `ServiceFailureException`, which is the one triggered by the current issue.